content/en/blog/news/first-post/index.md

```diff
@@ -3,15 +3,126 @@ date: 2024-02-28
+> 
+
+First generation serverless runtimes (such as AWS Lambda, Google Cloud Functions, Azure Functions, and their Kubernetes counterparts OpenWhisk and KNative) are all based on the principle of running a VM or container per function. While this afforded flexibility for application developers, complexity was introduced to platform engineers as neither compute type is designed to start quickly. Platform engineers operating these serverless platforms became responsible for an elaborate dance of pre-warming compute capacity and loading a workload just-in-time, making for a difficult tradeoff in cold start performance and cost. The result? A slow and inefficient first generation of Serverless.
+
+To illustrate this point, below are all the steps required to start a Kubernetes Pod:
+
+| Phase | Description |
+| --- | --- |
+| Kube init | Kubernetes schedules a Pod to a node, the node then prepares to run the Pod |
+| Image pull (cacheable) | The node pulls the container images from a remote repository to a local cache when needed |
+| Image mount | The node prepares the container images and mounts them |
+| Container start | The node configures and starts the Pod's containers and sidecars |
+| Application load | Each container loads application code and libraries, and initializes application memory. |
+| Application initialization | Each container performs application-specific init code (entry-point, setting up telemetry, validating connections to external dependencies, etc.) |
+| Pod ready | The Pod responds to the asynchronous readiness probe and signals that it is now ready |
+| Service ready | The Pod gets added to the list of active endpoints for a Service and can start receiving traffic |
+
+Circumventing this process, and the amount of duplication that occurs on every replica even if they are running on the same physical hardware, is where the [Spin][spin] WebAssembly Runtime and SpinKube shines.
+
+## Why SpinKube?
+
+SpinKube solves or mitigates many of the issues commonly associated with deploying, scaling, and operating Serverless applications.
+
+The foundational work that underpins the majority of this comes from removing the Container. Spin Applications are primarily distributed as OCI Artifacts, rather than traditional container images, meaning that you only ship your compiled application and its assets - without system dependencies.
+
+To execute those applications we have a [runwasi](https://github.com/containerd/runwasi) based [containerd-shim-spin](https://github.com/spinkube/containerd-shim-spin/) that takes a WebAssembly artifact, [pre-compiles it for the specific architecture](https://github.com/spinkube/containerd-shim-spin/pull/32) (caching the result in the containerd content store), and is then ready to service requests with [sub-millisecond startup times](https://fermyon.github.io/spin-benchmarks/criterion/reports/spin-executor_sleep-1ms/index.html) - no matter how long your application has been sitting idle.
```

Comment:
nit: technically it takes an entire Spin application, which can have several Wasm components; which you could refer to as a "WebAssembly artifact" if you want..